apps/frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,22 @@
+/// <reference types="vite-plugin-svgr/client" />
+
+import { type FC } from "react";
+
+import Eye from "~/assets/images/icons/eye.svg?react";
+import StrikedEye from "~/assets/images/icons/striked-eye.svg?react";
+
+import { mapIconNameToSvg } from "./helpers/helpers.js";
+import { type IconName, type Icon as Properties } from "./types/types.js";
+
+const iconNameToSvg = mapIconNameToSvg<IconName>({
```

Comment:
This map should be in `icon/libs/maps/icon-name-to-svg.map.ts`, also we don't need `mapIconNameToSvg` mapper.

Reply:
I've tried to implement what @GvoFor adviced, can you check it out?